applications/RomApplication/python_scripts/rom_manager.py

```diff
@@ -154,6 +154,52 @@ def Test(self, mu_test=None):
 
 
 
+    def RunFOM(self, mu_run=None):
+        if mu_run is None:
```

Comment:
```suggestion
        if not mu_run:
```
I think it's more pythonic this way.

Reply:
In the past, when using `not mu_run`, sometime I came across the error 
    ValueError: The truth value of an array with more than one element is ambiguous.
On the other hand, `mu_run is None` always works independently of the type of mu_run

I must recognize that given the type of `mu` (either a Python list or None), the suggested code will always work. 

I would stick to the less Pythonic, yet more English and robust expression. But if you insist I will change it across the script